src/components/Header/AuxiliaryDropdown.tsx

```diff
@@ -49,6 +50,11 @@ export const AuxiliaryDropdown = () => {
       text: "Project Patches",
       onClick: () => sendEvent({ name: "Click Project Patches Link" }),
     },
+    {
```

Comment:
Any thoughts on putting Project Settings last in the list? It feels like it may be the least-frequently accessed of the menu items.

Reply:
That sgtm i'll reorder it!